code/game/objects/structures/hand_cart.dm

```diff
@@ -0,0 +1,60 @@
+/obj/structure/hand_cart
+	name = "handcart"
+	desc = "A wheeled cart used to make heavy things less difficult to move through the power of lever-arm and the wheel."
+	icon = 'icons/obj/structures/handcart.dmi'
+	icon_state = "cart"
+	anchored = 0
+	density = 1
+	atom_flags = ATOM_FLAG_WHEELED
+	material_alteration = MAT_FLAG_ALTERATION_NAME | MAT_FLAG_ALTERATION_COLOR
+	material = /decl/material/solid/metal/steel
+	w_class = ITEM_SIZE_NO_CONTAINER
+	var/pixel_y_offset = 4
+	var/pixel_x_offset = 4
+	var/atom/movable/C
```

Comment:
Would probably be better to use mob buckling for this effect, similar to what rollerbeds do.

Reply:
Buckling only works for mobs currently, nvm.